stingray/events.py

```diff
@@ -709,10 +774,76 @@ def to_astropy_table(self):
     @staticmethod
     def from_astropy_table(ts):
```

Comment:
Same as above: docstring with requirements for `TimeSeries` object and what you'd typically expect to find in that object in terms of data and header info